tests/sanity/test/explorer/utils.js

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
I think this is same file as `tests/sanity/test/apiml/utils.js`, no need to create new one, we can just reuse the other one.